routes/api/contacts.js

```diff
@@ -1,25 +1,137 @@
-const express = require('express')
+const express = require("express");
+const { v4: uuidv4 } = require("uuid");
 
-const router = express.Router()
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const validateContact = require("../../utils/validator");
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const router = express.Router();
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+// GET contacts list =================================
+router.get("/contacts", async (_, res) => {
+  try {
+    const contacts = await listContacts();
+    if (contacts.length > 0) {
+      res
+        .status(200)
+        .json({ status: "200", message: "Success", data: { contacts } })
+    } else {
+      res
+        .status(404)
+        .json({ status: "404", message: "Contacts not found" });
+    }
```

Comment:
```suggestion
      res
        .status(200)
        .json({ status: "200", message: "Success", data: { contacts } })
```
Nie ma potrzeby takich variacji, generalnie jeśli nie ma błędu to zwracasz tablicę, pusta czy nie jest ok. To tylko sugestia w celu uproszczenia kodu :)

Reply:
Ok. Chociaż czytałam trochę o tych kodach odpowiedzi i znalazłam różne zdania na temat 'pustych' odpowiedzi. Wtedy zrobiłam tego 'if'a (tylko z kodem 204, żeby dać znać, że pytanie rzeczywiście jest dobrze obsłużone, ale w odpowiedzi jest 'pusto'. Na koniec jednak zmieniłam to 204 na 404. 
Ale przyjmuję do wiadomości, że w takim przypadku zwraca się po prostu OK z kodem 200 :)
Poprawione.